encord/user_client.py

```diff
@@ -13,9 +13,7 @@
 from __future__ import annotations
 
 import base64
-import json
```

Comment:
Worth remarking that the reason that the SDK doesn't tidy up imports is to allow including imports to ensure we maintain the same interface.
Removal of these libraries all good ofc.

Reply:
Yes, I deleted them based on that they belong to the standard library, so there is no intention to call them via the `encord` module.